src/Analysis/GWASResults/Components/Diagrams/lib/pheweb_plots.js

```diff
@@ -105,6 +105,29 @@ function create_gwas_plot(variant_bins, unbinned_variants_in, manhattan_plot_con
         return chrom_offsets[variant.chrom] + variant.pos;
     }
 
+    function escape_invalid_characters(value) {
+        // escape any characters that are invalid in a svg field:
+        const result = value.replace(/[<>&'"]/g, function(character) {
+            switch (character) {
+                case '<': return '&lt;';
+                case '>': return '&gt;';
+                case '&': return '&amp;';
+                case '\'': return '&apos;';
+                case '"': return '&quot;';
```

Comment:
Just a thought, are there other characters that could be problematic in the future? Also instead of manually doing this, consider using Lodash escape function (we already have the lodash library available for this code base)

### _escape
https://lodash.com/docs/4.17.15#escape

### _escapeRegExp
[_.escapeRegExp([string=''])](https://lodash.com/docs/4.17.15#escapeRegExp)

Reply:
done: https://github.com/uc-cdis/data-portal/compare/5c30537e0687c9b5a64d185bd78a5cd80578493d..6b7468a99e0399dc3c7d5dab03aaab9164684c9d